contributing/code_of_conduct/reporting_guidelines.rst

```diff
@@ -0,0 +1,99 @@
+Reporting Guidelines
+====================
+
+If you believe someone is violating the Code of Conduct we ask that you report
+it to the :doc:`enforcement team </contributing/code_of_conduct/enforcement_team>`
+by emailing, twitter, in person or any way you see fit.
+
+**All reports will be kept confidential.** The privacy of everyone included in
+the report is of our highest concern. Second to privacy there is transparency.
+After every report we will determine if a public statement should be made. If
+that's the case, the identities of all victims, reporters, and the accused will
+remain confidential unless those individuals instruct us otherwise. The details
+of the incident may also be generalized.
+
+If you believe anyone is in physical danger, please notify appropriate emergency
+services first by calling the relevant local authorities. If you are unsure what
+service or agency is appropriate to contact, include this in your report and we
+will attempt to notify them.
+
+In your report please include:
+
+  * Your contact info for follow-up contact.
+  * Names (legal, nicknames, or pseudonyms) of any individuals involved.
+  * If there were other witnesses besides you, please try to include them as well.
+  * When and where the incident occurred. Please be as specific as possible.
+  * Your account of what occurred.
```

Comment:
This sentence feels strange as a foreign english speaker, is `account` here meaning `point of view`?

Reply:
as account is being used here it means `a report or description of an event or experience.`

but if its confusing maybe lets replace `account` with `description` ?

Reply:
`description` is probably good as well.